research/minigo/README.md

```diff
@@ -56,7 +56,9 @@ parameterized layers respectively for the residual tower, plus an additional 2
 layers for the policy head and 3 layers for the value head.
 
 ## Getting Started
-Please follow the [instructions](https://github.com/tensorflow/minigo/blob/master/README.md#getting-started) in original Minigo repo to set up the environment.
+This project assumes you have virtualenv, TensorFlow (>= 1.5) and two other Go-related
+packages pygtp(>=0.4) and sgf (==0.5).
```

Comment:
Do we need pygtp in the end?

Reply:
I roughly checked all other py files, and it looks like some of them (coords, sgf, and others) import gtp as a module. I will dig more to see if it's necessary. Will remove it in next PR if it's redundant.